webui/tests/e2e-cluster/Makefile

```diff
@@ -0,0 +1,37 @@
+.PHONY: clean
+clean:
+	rm -rf build/
+
+.PHONY: start-db
+start-db: stop-db
+	# TODO make the db port configurable
+	docker-compose up -d
+
+.PHONY: stop-db
+stop-db:
+	docker-compose down
+
+.PHONY: psql
+psql:
+	docker exec -it \
+		tools_determined-db_1 \
+		psql determined -U postgres
+
+.PHONY: prep-root
+prep-root:
+	@rm -rf build
+	@mkdir -p build/webui/docs
+	@mkdir -p build/webui/elm
+	@mkdir -p build/webui/react
+	@mkdir -p build/wheels/
+	@ln -s $(abspath $(PROJECT_ROOT)/master/static) build/
+	@ln -s $(abspath $(PROJECT_ROOT)/docs/site/html/*) build/webui/docs
+	@ln -s $(abspath $(PROJECT_ROOT)/webui/react/build/*) build/webui/react
+	@ln -s $(abspath $(PROJECT_ROOT)/webui/elm/public) build/webui/elm
+	@ln -s $(abspath $(PROJECT_ROOT)/common/dist/*.whl) build/wheels/
+	@ln -s $(abspath $(PROJECT_ROOT)/cli/dist/*.whl) build/wheels/
+	@ln -s $(abspath $(PROJECT_ROOT)/harness/dist/*.whl) build/wheels/
```

Comment:
This didn't work for me and it turns out it was because I don't have `$PROJECT_ROOT` defined. I couldn't find any other references to this env anywhere in the project. Should this be defined? or is it specific to your setup?

Reply:
setting the env allowed the `make test` to run great

Reply:
looking at the original tools `Makefile` @ `webui/tests/tools/Makefile` it looks like they used a relative path instead, since `abspath` converts it back to an absolute path. For example for the first `@ln` line would be this instead:
```
# since we are in `webui/tests/e2e-cluster`
@ln -s $(abspath ../../../master/static) build/
```

Reply:
yeah.. sorry my bad you must have ran into a bunch of errors not knowing where they were coming from..! I forgot I set it there :man_facepalming:

Reply:
no worries, it wasn't too bad to diagnose and it was good that it forced me to look through more closely.